### PR TITLE
Finally introduce unit tests

### PR DESCRIPTION
**Pull request by bendoh**
## Summary

Get a harness for unit tests as given by the wp-cli plugin test tool. Add a single passing test.
## Risk
- [X] Trivial
- [ ] Low
- [ ] Medium
- [ ] High
## How to Test

Check out and run the tests! Usage is detailed in `README.md`

### CODE REVIEW

File: bin/install-wp-tests.sh [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+		if [ $WP_VERSION == 'latest' ]; then
+			local ARCHIVE_NAME='latest'
+		else
+			local ARCHIVE_NAME="wordpress-$WP_VERSION"
+		fi
+		download https://wordpress.org/${ARCHIVE_NAME}.tar.gz  /tmp/wordpress.tar.gz
+		tar --strip-components=1 -zxmf /tmp/wordpress.tar.gz -C $WP_CORE_DIR
+	fi
+
+	download https://raw.github.com/markoheijnen/wp-mysqli/master/db.php $WP_CORE_DIR/wp-content/db.php
```
Comment (marcaddeo) [inline bin/install-wp-tests.sh:66]: I'm curious, what's this for?

**Comment by marcaddeo** [comment]
So how does this work? Does this run a webserver at all? Or does it just rely on the database for tests?

Just curiosity! Otherwise looks good! /approved

Oh, maybe add a "Build: Passing" badge to the README?

File: bin/install-wp-tests.sh (hunk 1 above)
Comment (bendoh) [inline bin/install-wp-tests.sh:66]: Unsure about this specific thing, it's just part of the boilerplate. I guess I could keep it out of the repository and install it always using `wp scaffold`, but meh.

Comment (bendoh) [inline bin/install-wp-tests.sh:66]: It's a script that's generated by `wp scaffold plugin-tests` along with `phpunit.xml`, `.travis.yml` and `tests/bootstrap.php`, so i didn't really write any of it. Appears to install a sample database?

**Comment by bendoh** [comment]
Good idea, since we're getting travis anyway...

**Comment by marcaddeo** [comment]
Look's like you gotta increase the min php version in `.travis.yml`. Idk if you can add such aa requirement explicitly to the plugin though.

**Comment by bendoh** [comment]
Yay! It passes. /approved

**Pull request merged**